src/scmdata/run.py

```diff
@@ -478,6 +482,9 @@ def _init_timeseries(
 
             (_df, _meta) = _read_file(data, required_cols=self.required_cols, **kwargs)
 
+        if _df.index.duplicated().any():
```

Comment:
Yikes did we not have any coverage of this error previously

Reply:
Yep, fortunately I think it's a hard one to muck up (you have to really try to duplicate data)